python/semantic_kernel/agents/open_ai/openai_responses_agent.py

```diff
@@ -190,7 +191,6 @@ def id(self) -> str | None:
         """Get the thread ID."""
         return self.response_id
 
-    @override
```

Comment:
`_create` is an abstract method in the base class - we should leave the `@override` decorator.